### DISCUSSION TITLE
https://github.com/Abhimanyu9555657/

### DISCUSSION THREAD

**Discussion by Abhimanyu9555657** [learndevopsonline/learndevopsonline#1794] [Q&A]
Hi Sir,

When iam adding a Tag and the jobs are getting stuck.
ERROR: There are no nodes with the label ‘[workstation](http://54.172.101.123:8080/label/workstation/)’

![image](https://github.com/learndevopsonline/learndevopsonline/assets/153291922/c99fb2ab-c18b-4d38-82d1-eadc28c877a5)

and same is the situation with TAG (Test1)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/153291922/c8292292-f600-47ad-936a-157e0210fabe)

**Comment by chaithanya1812** [comment]
Error:  No nodes with label found workstation. You can name any label name not only workstation,
But In pipelines we need to use right label. 
pipeline{
agent {
label  'your-defined-label' 
}
}
pls go through once, If issue is not sloved pls join dissicussion link.

**Reply by chaithanya1812** [reply]
please tell your jenkins-credentials

**Reply by Abhimanyu9555657** [reply]
Username: admin
Password : admin123

**Comment by chaithanya1812** [comment]
actually you didn't gave a label name, This was the error. Now I add  the label worktsation , it's working fine check once
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/e0b0cded-38b0-409d-befe-84fc23fb4974)

**Reply by Abhimanyu9555657** [reply]
Thanks Sir Let me go back to Video 53 and start once again
